Add xsltproc to build container

adrianghc opened the pull request:
From the xsltproc website:

> xsltproc is a command line tool for applying XSLT stylesheets to XML documents. It is part of libxslt, the XSLT C library for GNOME. While it was developed as part of the GNOME project, it can operate independently of the GNOME desktop.

This is necessary e.g. for https://github.com/RIOT-OS/RIOT/pull/7903 and possibly other packages in the future.

gebart commented on Dockerfile:
```diff
@@ -71,6 +71,7 @@ RUN \
         unzip \
         vim-common \
         wget \
+		xsltproc \
```
Fix the indentation to match the line above

adrianghc commented:
@gebart From "apt-cache show xsltproc" on Ubuntu:

>Package: xsltproc
Installed-Size: 145
Architecture: amd64
Source: libxslt
Version: 1.1.28-2.1
Depends: libc6 (>= 2.4), libxml2 (>= 2.7.4), libxslt1.1 (>= 1.1.25)
Size: 13362

Does this help?

cladmi commented:
Please squash both commits and say why you add `xsltproc`. It will help removing it if it is not needed anymore.

From the build size difference docker says the same rounded size between this rebased and master `3.57GB`

adrianghc commented:
@cladmi Would you like me to say that in the commit message? How much detail is good for you?

adrianghc commented:
@cladmi I've rebased my branch to your master, squashed both commits and added a short description in the commit message. Please let me know if it looks alright to you now.

cladmi commented:
Please re-rebase before merging, it has some conflicts again that github cannot resolve.

@gebart do you agree to merge ?

cladmi commented:
@adrianghc can you please rebase

adrianghc commented:
@gebart @cladmi I didn't see any conflicts but anyway I rebased and I think it should be fine now, apart from Travis CI failing with an error that seems unrelated to my changes and seems to be occurring on other builds as well since 10 days ago.

> xz: (stdin): File format not recognized

aabadie commented:
> apart from Travis CI failing with an error that seems unrelated to my changes and seems to be occurring on other builds as well since 10 days ago

the build failure is fixed by #61.

aabadie commented:
I think so. #61 should be merged fast, since it's a one line fix.

adrianghc commented:
@aabadie Well, so is this one and it's taking a while. 😅

miri64 commented:
Because travis should retest this with current master as base.

miri64 commented:
![image](https://user-images.githubusercontent.com/675644/48711892-ab26a780-ec0c-11e8-8250-fcec81a4befe.png)

(just needs a rebase)

adrianghc commented:
@miri64 Rebase is done.

miri64 commented:
I leave merging to the ACKing parties :-) @cladmi?

cladmi commented:
@adrianghc thanks for following all the rebase requests

@miri64 I tried the "update branch" button on one of @bergzand PR and it messed up completely the testing after, so a bit reluctant to use it https://github.com/RIOT-OS/RIOT/pull/10185 both murdock and travis could not rebase anymore… :/

cladmi commented:
Or to be precise, our `static-tests` could not rebase it anymore.

miri64 commented:
> Or to be precise, our static-tests could not rebase it anymore.

Then we need to fix those. Rebase after merge should be possible AFAIK.
